packages/tasks/src/model-libraries.ts

```diff
@@ -1,52 +1,333 @@
+		docsUrl: "https://huggingface.co/docs/hub/stanza",
+		snippets: snippets.stanza,
+		filter: true,
+	},
+	tensorflowtts: {
+		btnLabel: "TensorFlowTTS",
+		repoName: "TensorFlowTTS",
+		repoUrl: "https://github.com/TensorSpeech/TensorFlowTTS",
+		snippets: snippets.tensorflowtts,
+	},
+	timm: {
+		btnLabel: "timm",
+		repoName: "pytorch-image-models",
+		repoUrl: "https://github.com/rwightman/pytorch-image-models",
+		docsUrl: "https://huggingface.co/docs/hub/timm",
+		snippets: snippets.timm,
+		filter: true,
+	},
+	transformers: {
+		btnLabel: "Transformers",
+		repoName: "🤗/transformers",
+		repoUrl: "https://github.com/huggingface/transformers",
+		docsUrl: "https://huggingface.co/docs/hub/transformers",
+		snippets: snippets.transformers,
+		filter: true,
+	},
+	"transformers.js": {
+		btnLabel: "Transformers.js",
+		repoName: "transformers.js",
+		repoUrl: "https://github.com/xenova/transformers.js",
+		docsUrl: "https://huggingface.co/docs/hub/transformers-js",
+		snippets: snippets.transformersJS,
+		filter: true,
+	},
+	fasttext: {
+		btnLabel: "fastText",
+		repoName: "fastText",
+		repoUrl: "https://fasttext.cc/",
+		snippets: snippets.fasttext,
+		filter: true,
+	},
+	"sample-factory": {
+		btnLabel: "sample-factory",
+		repoName: "sample-factory",
+		repoUrl: "https://github.com/alex-petrenko/sample-factory",
+		docsUrl: "https://huggingface.co/docs/hub/sample-factory",
+		snippets: snippets.sampleFactory,
+		filter: true,
+	},
+	"stable-baselines3": {
+		btnLabel: "stable-baselines3",
+		repoName: "stable-baselines3",
+		repoUrl: "https://github.com/huggingface/huggingface_sb3",
+		docsUrl: "https://huggingface.co/docs/hub/stable-baselines3",
+		snippets: snippets.stableBaselines3,
+		filter: true,
+	},
+	mindspore: {
+		btnLabel: "MindSpore",
+		repoName: "mindspore",
+		repoUrl: "https://github.com/mindspore-ai/mindspore",
+	},
+	"ml-agents": {
+		btnLabel: "ml-agents",
+		repoName: "ml-agents",
+		repoUrl: "https://github.com/Unity-Technologies/ml-agents",
+		docsUrl: "https://huggingface.co/docs/hub/ml-agents",
+		snippets: snippets.mlAgents,
+		filter: true,
+	},
+	"unity-sentis": {
+		btnLabel: "unity-sentis",
+		repoName: "unity-sentis",
+		repoUrl: "https://github.com/Unity-Technologies/sentis-samples",
+		snippets: snippets.sentis,
+		filter: true,
+	},
+	pythae: {
+		btnLabel: "pythae",
+		repoName: "pythae",
+		repoUrl: "https://github.com/clementchadebec/benchmark_VAE",
+		snippets: snippets.pythae,
+		filter: true,
+	},
+} satisfies Record<string, LibraryUiElement>;
+
+export type ModelLibraryKey = keyof typeof MODEL_LIBRARIES_UI_ELEMENTS;
 
-export const ALL_MODEL_LIBRARY_KEYS = Object.keys(ModelLibrary) as ModelLibraryKey[];
+export const ALL_MODEL_LIBRARY_KEYS = Object.keys(MODEL_LIBRARIES_UI_ELEMENTS) as ModelLibraryKey[];
 
-export const ALL_DISPLAY_MODEL_LIBRARY_KEYS = ALL_MODEL_LIBRARY_KEYS.filter(
-	(k) => !["doctr", "k2", "mindspore", "tensorflowtts"].includes(k)
-);
```

Comment:
there was already this @osanseviero but i wrapped it inside the main dict

Note that i used an opt-in "filter: true" which seems friendlier to me

The elastic queries i will add in a second PR

Reply:
Awesome, thanks a lot!

Reply:
Was `ALL_MODEL_LIBRARY_KEYS` being used? They were still shown :thinking:

Reply:
yes, looks like `ALL_DISPLAY_MODEL_LIBRARY_KEYS` (the filtered one) was only used in the suggestion dropdown in the GUI editor:

<img width="945" alt="image" src="https://github.com/huggingface/huggingface.js/assets/326577/382ad68a-5d83-41da-92a8-5c754486f021">

Reply:
Ok, we need a follow up PR so `ALL_DISPLAY_MODEL_LIBRARY_KEYS` is also used to control the filters at left of hf.co/models